stripper/ze_pirates_port_royal_v5_6.cfg

```diff
@@ -1,3 +1,27 @@
+;Adjust level 2 cannon bossfight moving speed as recent server-side fixes caused the cannon to move unusually fast resulting in close-calls every single time.
```

Comment:
Should really specifically mention the parented trigger fix here, rather than being vague.